sumologic/sumologic.py

```diff
@@ -217,4 +246,59 @@ def install_app(self, app_id, content):
         return self.post('/apps/%s/install' % (app_id), params=content)
 
     def check_app_install_status(self, job_id):
-        return self.get('/apps/install/%s/status' % job_id)
\ No newline at end of file
+        return self.get('/apps/install/%s/status' % job_id)
+
+    def export_content(self, content_id):
+        return self.post('/content/%s/export' % content_id, params=content_id, version='v2')
```

Comment:
do we need params also? I think it should work with params="" as well.

Reply:
Good spotting, you're correct with leaving them blank. I've committed the change